### PR TITLE
Test: Check attach functionality

### PR DESCRIPTION
**Pull request by GabyCT**
This will verify that attach is working properly

Signed-off-by: Cervantes Tellez, Gabriela <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-failed

File: tests/integration/docker/attach.bats [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+setup() {
+	source $SRC/test-common.bash
+	clean_docker_ps
+	runtime_docker
+}
+
+@test "Check attach functionality" {
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 5 && echo 'Hello, World!'"
+	$DOCKER_EXE attach containertest > result_file
+	[ -s result_file ]
```
Comment (jcvenegas) [inline tests/integration/docker/attach.bats]: @GabyCT what is suppose to have that file? what about check the output as well to verify is what you want

File: tests/integration/docker/attach.bats [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+SRC="${BATS_TEST_DIRNAME}/../../lib/"
+
+setup() {
+	source $SRC/test-common.bash
+	clean_docker_ps
+	runtime_docker
+}
+
+@test "Check attach functionality" {
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 5 && echo 'Hello, World!'"
```
Comment (jcvenegas) [inline tests/integration/docker/attach.bats]: It is not needed a tear down  function? or use docker run --rm remove the last container

File: tests/integration/docker/attach.bats (hunk 1 above)
Comment (GabyCT) [inline tests/integration/docker/attach.bats]: @jcvenegas  the file should have the hello world and what I am doing at the end is checking that the file is not empty

File: tests/integration/docker/attach.bats (hunk 2 above)
Comment (GabyCT) [inline tests/integration/docker/attach.bats]: At the beginning of each of our integration tests we are performing a clean in the environment

File: tests/integration/docker/attach.bats (hunk 1 above)
Comment (GabyCT) [inline tests/integration/docker/attach.bats]: @jcvenegas you can not combine -d with --rm

File: tests/integration/docker/attach.bats (hunk 2 above)
Comment (amshinde) [inline tests/integration/docker/attach.bats]: I would prefer a teardown function to ensure that the docker instances are cleaned up after running an individual test. Either that or --rm

File: tests/integration/docker/attach.bats (hunk 1 above)
Comment (amshinde) [inline tests/integration/docker/attach.bats]: @GabyCT A better check would be to verify actual contents of the file.

**Comment by sameo** [comment]
LGTM

**Comment by jcvenegas** [comment]
lgtm

**Comment by jcvenegas** [comment]
lgtm

**Comment by jcvenegas** [comment]
LGTM

**Pull request merged**